Metrics-based SLOs (GA 8/18) (#2725)
SumoLogic/sumologic-documentation: blog-service/2023-08-18-metrics-slos.md

---
title: Metrics-Based SLOs (Reliability Management)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - slo
  - observability
  - metrics
  - operators
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> You can now create SLOs from the **Metrics** page and use [metrics operators](/docs/metrics/metrics-operators) in these SLO queries. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).

Reviewer: ```suggestion
You can now create SLOs from the **Metrics** page and use [metrics operators](/docs/metrics/metrics-operators) in these SLO queries. The metrics query specified in your SLO should have a quantization specified after the selector. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).
```

> You can now create SLOs from the **Metrics** page with [metrics operators](/docs/metrics/metrics-operators) in SLO queries. The metrics query specified in your SLO should have a quantization specified after the selector. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).

Reviewer 2: ```suggestion
You can now use [metrics operators](/docs/metrics/metrics-operators) in SLO queries. The metrics query specified in your SLO should have a quantization specified after the selector. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).
```

> You can now create SLOs from the **Metrics** page and use [metrics operators](/docs/metrics/metrics-operators) in these SLO queries. The metrics query specified in your SLO should have a quantization specified after the selector. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).

Reviewer 3: ```suggestion
You can now create SLOs from the **Metrics** page with [metrics operators](/docs/metrics/metrics-operators) in SLO queries. The metrics query specified in your SLO should have a quantization specified after the selector. You can specify one or more operators in the query for SLO. [Learn more](/docs/observability/reliability-management-slo/create-slo/#create-an-slo-from-metrics-page).
```
